api/store/mongo/device_store_tags_test.go

```diff
@@ -13,7 +13,7 @@ import (
 func TestDeviceCreateTag(t *testing.T) {
 	data := initData()
 
-	db := dbtest.DBServer{}
+	db := dbtest.DBServer{Replicaset: true}
```

Comment:
In the past, we had support for use with and without replicaset. However, since a few releases ago, we have used replicaset in all deployments, and as a consequence, we should use it by default and avoid touching all test code.